nav2_tasks/include/nav2_tasks/is_stuck_condition.hpp

```diff
@@ -91,14 +59,33 @@ class IsStuckCondition : public BT::ConditionNode, public rclcpp::Node
 
   ~IsStuckCondition()
   {
+    RCLCPP_DEBUG(this->get_logger(), "Shutting down IsStuckCondition BT node");
     stopWorkerThread();
   }
 
+  void onOdomReceived(const typename nav_msgs::msg::Odometry::SharedPtr msg)
```

Comment:
This method is already in the robot class. Please use robot class.

Reply:
@mhpanah as discussed this will be part of the larger effort to move all robot specific algorithms to the robot class. For now I'll merge as is.